ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -200,6 +200,7 @@ const getFilteredTransactionGroupsAllChains = (
 const groupTransactionsByDate = (
   transactionGroups,
   getTransactionTimestamp,
+  shouldSort = true,
```

Comment:
[nit] option to make this an optional param 
```suggestion
  { shouldSort = true } = {},
```
and update below instance with shouldSort: false below